pkg/kubelet/volumemanager/populator/desired_state_of_world_populator.go

```diff
@@ -434,14 +442,26 @@ func volumeRequiresFSResize(pvc *v1.PersistentVolumeClaim, pv *v1.PersistentVolu
 }
 
 // podPreviouslyProcessed returns true if the volumes for this pod have already
-// been processed by the populator
+// been processed/reprocessed by the populator. Otherwise, the volumes for this pod need to
+// be reprocessed.
 func (dswp *desiredStateOfWorldPopulator) podPreviouslyProcessed(
 	podName volumetypes.UniquePodName) bool {
 	dswp.pods.RLock()
 	defer dswp.pods.RUnlock()
 
-	_, exists := dswp.pods.processedPods[podName]
-	return exists
+	processed := dswp.pods.processedPods[podName]
+	return processed
+}
+
+// podHasBeenProcessedOnce returns true if the pod has been processed by the popoulator
+// at least once.
+func (dswp *desiredStateOfWorldPopulator) podHasBeenProcessedOnce(
```

Comment:
This is more like whether pod has been seen once or not by DSWP rather than processed.Can we change the name?

Reply:
@gnufied 
The func has been renamed